docs/operations/clean-metadata-store.md

```diff
@@ -0,0 +1,104 @@
+---
+id: clean-metadata-store
+title: "Automated cleanup for metadata records related to deleted datasources"
```

Comment:
It is not just datasources but other entities as well (such as compaction config, rules, etc.). For example, you might only ever have a single datasource, but you continuously create and delete many many compaction rules for that single datasource.